src/main/java/cd/go/framework/ldap/SearchResultLimitExceededException.java

```diff
@@ -0,0 +1,22 @@
+package cd.go.framework.ldap;
+
+import java.text.MessageFormat;
+
+public class SearchResultLimitExceededException extends RuntimeException {
+    private final int maxResult;
+    private final String searchBase;
+
+    public SearchResultLimitExceededException(int maxResult, String searchBase) {
+        super(MessageFormat.format("Search results limit exceeded. Expected search to return {0} results but found more in search base {1}.", maxResult, searchBase));
```

Comment:
This exception is only logged, right? Never shown to the user?

Reply:
Yes. this will be logged.